src/Orleans.Core/Core/DeadlockException.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+
+namespace Orleans.Runtime
+{
+    /// <summary>
+    /// Signifies that a silo has detected a deadlock / loop in a call graph.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// Deadlock detection is not enabled by default in Orleans silos, 
+    /// because it introduces some extra overhead in call handling.
+    /// </para>
+    /// <para>
+    /// There are some constraints on the types of deadlock that can currently be detected 
+    /// by Orleans silos.
+    /// </para>
+    /// </remarks>
+    [Serializable]
+    public class DeadlockException : OrleansException
```

Comment:
why not move this one to the Exceptions folder with the rest?

Reply:
Forgot to remove it